### ISSUE TITLE
keyboard nav - "j" on first msg broken

### ISSUE THREAD

**Issue by freste** [leggett/simplify#639]
3.0.2
new info - only happens with Bundling turned on. (And I'm guessing one or more bundles in the list.)

It appears that if you're viewing the first msg in a list (e.g. "Primary") and hit "k", nothing happens. I believe it used to (or at least I expect it to) go back to displaying the list of msgs, e.g. same as "Esc" would.

Also of interest - after pressing "k", "j" also no longer works, will not go to the next msg.
Esc does go back to the list.

Console shows this once:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'classList')
    at HTMLBodyElement.onKeydown (simplifyGmail.js:7)

```
And this on every subsequent j/k:
```
simplifyGmail.js:7 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'style')
    at HTMLBodyElement.onKeydown (simplifyGmail.js:7)
```

**Comment by freste** [comment]
Sorry, 3.0.2

**Comment by leggett** [comment]
I thought I replied to this but apparently not -- thanks for the detailed report and the console logs. Super helpful. I hope to fix this in the next update -- will close this issue when I do.

**Comment by leggett** [comment]
And to confirm -- yes, this is related to bundles. Once you close the message and re-open it, J/K works again.

**Comment by leggett** [comment]
Fix about to land in v3.0.5
